connect/README.md

```diff
@@ -0,0 +1,64 @@
+[//]: # (This readme is optimized for inline rustdoc, if some links don't work, they will when included in lib.rs)
+
+# Connect
+
+The connect module of librespot. Provides the option to create your own connect device
+and stream to it like any other official spotify client.
+
+The [`Spirc`] is the entrypoint to creating your own connect device. It can be
+configured with the given [`ConnectConfig`] options and requires some additional data
+to start up the device. 
+
+When creating a new [`Spirc`] it returns two items. The [`Spirc`] itself, which is can 
+be used as to control the local connect device. And a [`Future`](std::future::Future), 
+lets name it `SpircTask`, that starts and executes the event loop of the connect device 
+when awaited.
+
+To get an understanding how to handle the `SpircTask`, it is recommended to take look 
+at the code of the `librespot` binary. As the [`src/main.rs`](https://github.com/librespot-org/librespot/blob/dev/src/main.rs#L1943) 
```

Comment:
Maybe link this to a specific commit, so we can be sure the reference to this line remains as we expect?

Reply:
Could we extract it into a new example?

Reply:
I added a minimal compilable version here in the README (so that you have an example on docs.rs). But for a greater picture I just point to a working version, which is as currently the binary. 

But yeah in general an own example would be technically better... Yeah why not! I had the thought initially, but wanted to only provided an example in the README (probably laziness). Tho it's probably better to point to a static point, as the example will hopefully not change as the binary will.

Reply:
I was wondering if there is any rust tooling that takes examples and inserts them into rustdoc automatically when you publish the crate. I think Python has stuff like that. I did find found https://github.com/rust-lang/rust/issues/88791 which looks cool.

Reply:
Well actually there was already an example I didn't fully noticed so far. I updated it a bit and referenced it in the README now.